Formula/kat.rb

```diff
@@ -39,15 +46,30 @@ def install
       s.gsub! "-Wno-long-double", ""
     end
 
-    system "./build_boost.sh"
+    boost = Formula["boost"]
+    inreplace "lib/Makefile.am", "$(top_builddir)/deps/boost/build/lib", boost.opt_lib
+    inreplace ["src/Makefile.am", "tests/Makefile.am"] do |s|
+      s.gsub! "$(top_builddir)/deps/boost/build/lib", boost.opt_lib
+      s.gsub! "$(top_srcdir)/deps/boost/build/include", boost.opt_include
+    end
+
+    venv = virtualenv_create(libexec/"vendor", "python3")
+    venv.pip_install resources
+
+    xy = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python#{xy}/site-packages"
+
     system "./autogen.sh"
     system "./configure",
       "--disable-silent-rules",
-      "--disable-dependency-tracking",
       "--disable-pykat-install",
```

Comment:
```suggestion
```
I'm not totally sure about this, but @SeekingMeaning if we're going to install tabulate, shouldn't we install pykat?

Reply:
Hmm I'm not quite sure myself

Reply:
What's the reason for removing `"--disable-dependency-tracking",`?